src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -80,21 +82,17 @@
 
-"ExposureDetection_Explanation_Text_Off" = "Lorem Ipsum..";
+"ExposureDetection_Explanation_Subtitle_Inactive" = "This is how your risk level is calculated";
 
-"ExposureDetection_Explanation_Text_Unknown" = "Your risk of infection is unknown.";
+"ExposureDetection_Explanation_Text_Off" = "Your risk of infection is calculated from the risk identification data (duration and proximity) locally on your device. Your risk of infection cannot be seen by, or passed on to, anyone else.";
 
-"ExposureDetection_Explanation_Text_Inactive" = "You have a low risk of infection since no encounter to anyone diagnosed with COVID-19 has been logged, or your encounter was restricted to a short time and at a greater distance.\n\nTherefore your risk of infection has been ranked as low.\n\nThe risk of infection is calculated from the risk identification data (duration and proximity) locally on your device. Your risk of infection cannot be seen by, or passed on to, anyone else.";
+"ExposureDetection_Explanation_Text_Unknown" = "Since you have not activated risk identification for long enough, we could not calculate your risk of infection.\n\nYour risk of infection is calculated from the risk identification data (duration and proximity) locally on your device. Your risk of infection cannot be seen by, or passed on to, anyone else.";
 
 "ExposureDetection_Explanation_Text_Low" = "You have a low risk of infection since no encounter with a person diagnosed with COVID-19 has been logged, or your encounter was restricted to a short time and at a greater distance.\n\nTherefore your risk of infection has been ranked as low.\n\nThe risk of infection is calculated from the risk identification data (duration and proximity) locally on your device. Your risk of infection cannot be seen by, or passed on to, anyone else.";
 
-"ExposureDetection_Explanation_Text_High" = "You have an increased risk of infection because you were last exposed two days ago over a longer period of time and at close proximity to at least one person diagnosed with COVID-19.\n\nTherefore, your risk of infection has been ranked as increased.\n\nWhen you get home, please also avoid close contact with members of your family or household.";
-
-"ExposureDetection_MoreInformation" = "More information";
-
-"ExposureDetection_MoreInformation_URL" = "https://www.google.de";
+"ExposureDetection_Explanation_Text_High" = "You have an increased risk of infection because you were last exposed two days ago over a longer period of time and at close proximity to at least one person diagnosed with COVID-19.\n\nTherefore, your risk of infection has been ranked as increased.\nYour risk of infection is calculated from the risk identification data (duration and proximity) locally on your device. Your risk of infection cannot be seen by, or passed on to, anyone else.\nWhen you get home, please also avoid close contact with members of your family or household.";
 
 "ExposureDetection_Hotline_Number" = "+498007540002";
```

Comment:
Where is the hotline_text key?

Reply:
never mind, I think the hotline is gone from the exposure detection screen. That key can be deleted I guess. Same goes for the German one.

Reply:
removed